marshmallow/fields.py

```diff
@@ -306,6 +306,8 @@ def _deserialize(self, value, attr, data):
         :param value: The value to be deserialized.
         :param str attr: The attribute/key in `data` to be deserialized.
         :param dict data: The raw input data passed to the `Schema.load`.
+        :param bool|tuple partial: For nested schemas, the ``partial``
```

Comment:
So, should either `partial` or `**kwargs` appear in the signature?

Reply:
This feels like a project style decision so I didn't update it on my own.

My two cents: I think I can personally argue it both ways.  Adding it feels unnecessary from a client perspective.  It doesn't tell me anything about how I can directly use the function.  On the other hand, if I'm "using" the library by subclassing elements (which honestly feels like a typical use case), then I kind of want to know that the function will accept arbitrary keyword arguments.  The reason I didn't add it in the first place is that by the time I'm subclassing the function I'm already looking at the code so the documentation didn't seem as relevant.